api/client/trust.go

```diff
@@ -411,14 +416,17 @@ func (cli *DockerCli) trustedPush(repoInfo *registry.RepositoryInfo, tag string,
 	keys := ks.RootKeyStore().ListKeys()
 	var rootKey string
 
-	if len(keys) == 0 {
+	// TODO(dmcgowan): let user choose if multiple keys, right now just use first
+	for k := range keys {
```

Comment:
If there were multiple keys yes. The options would be to sort and choose the first or store the preferred key in a configuration file. Both of these are slightly larger changes, especially considering this will usually be at most 1. But either way this behavior should be clearly defined.